airflow/cli/commands/variable_command.py

```diff
@@ -65,7 +65,7 @@ def variables_import(args):
     if os.path.exists(args.file):
         _import_helper(args.file)
     else:
-        print("Missing variables file.")
+        raise SystemExit("Missing variables file.")
```

Comment:
```suggestion
        raise SystemExit("\n\tMissing variables file.")
```
If we want to be consistent... but to be honest I'm not sure if we need this "prefix".

Reply:
*Style* consistency will need more effort. For example:
- with or without prefix "\n\t" are varying among different sub-command-groups (the same raises in `user_command.py` are withOUT this "prefix").
- some CLI failure are directly throwing "original" Python exceptions.

In this PR I would dedicate on the exit status, if you don't mind.